src/Data.hs

```diff
@@ -32,18 +35,23 @@ instance ToRow User where
     toField . ki $ user,
     toField . registration_date $ user]
 
+instance MonadDb m => MonadDb (ReaderT r m)
```

Comment:
This is needed for the compiler to generate the MonadDb instance for ReaderT and ExceptT. Only this two are here because they are the two transformers we have in AppM but there could potentially be instances for other transformers.